carma-messenger-core/message/message/src/main/java/gov/dot/fhwa/saxton/carma/message/factory/DSRCMessageFactory.java

```diff
@@ -31,9 +31,11 @@
         case "MobilityPath":
             return new MobilityPathMessage(log, factory);
         case "MobilityResponse":
-            return new MobilityResponseMessage(log, factory);
+            return null;
```

Comment:
May be worth also making sure this node doesn't actually receive these message (by removing or changing the topic mappings that it subscribes to, as I could see this causing a NullPointerException at runtime.